app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -58,7 +55,7 @@ const Notes = () => {
       />
       {notes.length ? (
         <>
-          <SubHeader
+          {/* <SubHeader
             rightActionBlock={
               <Button
                 disabled={!selectedNoteIds.length}
```

Comment:
Go through the Slack thread very well and then only assign me the PRs.

Reply:
I have removed all the unnecessary comments from PR. Pardon me again for my inability to notice it.